roles/teamcity-agent/tasks/main.yml

```
@@ -95,6 +95,12 @@
 - name: disable systemd as default ddns server
   shell: ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
 
+- name: set default java version # to get available versions run update-alternatives --list java
+  shell: |
+    update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+    update-alternatives --set javac /usr/lib/jvm/java-8-openjdk-amd64/bin/javac
```

There's a more specific variant of this command- [`update-java-alternatives`](http://manpages.ubuntu.com/manpages/bionic/man8/update-java-alternatives.8.html)! This means you only need to do one command, not two (in fact it sets the defaults for all the other Java commands too):


```suggestion
- name: set default java version # to get available versions run `update-java-alternatives --list`
  shell: |
    update-java-alternatives --set /usr/lib/jvm/java-1.8.0-openjdk-amd64
```

---

nice! Thanks Roberto